linux_mode/scripts/utils.py

```diff
@@ -0,0 +1,28 @@
+# imports
```

Comment:
Same for this file, let's merge it into a single file where all the gdb / dump logic is implemented.